staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1600,12 +1600,26 @@ type MigrationConfiguration struct {
 // DeveloperConfiguration holds developer options
 // +k8s:openapi-gen=true
 type DeveloperConfiguration struct {
-	FeatureGates           []string          `json:"featureGates,omitempty"`
-	LessPVCSpaceToleration int               `json:"pvcTolerateLessSpaceUpToPercent,omitempty"`
-	MemoryOvercommit       int               `json:"memoryOvercommit,omitempty"`
-	NodeSelectors          map[string]string `json:"nodeSelectors,omitempty"`
-	UseEmulation           bool              `json:"useEmulation,omitempty"`
-	CPUAllocationRatio     int               `json:"cpuAllocationRatio,omitempty"`
+	FeatureGates           []string            `json:"featureGates,omitempty"`
+	LessPVCSpaceToleration int                 `json:"pvcTolerateLessSpaceUpToPercent,omitempty"`
+	MemoryOvercommit       int                 `json:"memoryOvercommit,omitempty"`
+	NodeSelectors          map[string]string   `json:"nodeSelectors,omitempty"`
+	UseEmulation           bool                `json:"useEmulation,omitempty"`
+	CPUAllocationRatio     int                 `json:"cpuAllocationRatio,omitempty"`
+	DebugConfiguration     *DebugConfiguration `json:"debugConfiguration,omitempty"`
+}
+
+// DebugConfiguration sets various components to a debug mode
+// +k8s:openapi-gen=true
+type DebugConfiguration struct {
+	DebugAllVirtComponents bool `json:"debugAllVirtComponents,omitempty"`
+	DebugVirtApi           bool `json:"debugVirtApi,omitempty"`
+	DebugVirtControllers   bool `json:"debugVirtControllers,omitempty"`
+	DebugVirtLaunchers     bool `json:"debugVirtLaunchers,omitempty"`
+	DebugAllNodes          bool `json:"debugAllNodes,omitempty"`
```

Comment:
Using the term `Debug` here for libvirt/qemu debugging on nodes makes sense. :+1: